packages/next/src/client/app-index.tsx

```diff
@@ -23,6 +27,8 @@ import { MissingSlotContext } from '../shared/lib/app-router-context.shared-runt
 
 /// <reference types="react-dom/experimental" />
 
+const isReactOwnerStackEnabled = !!process.env.__NEXT_REACT_OWNER_STACK
```

Comment:
We decided to not flag it. We can still keep the feature flag to not make it look scary but internally, we can just check for `typeof React.captureOwnerStack === 'function'`

Reply:
Like even without experimental flag?